service/dap/server.go

```diff
@@ -1831,8 +1870,8 @@ func (s *Server) onEvaluateRequest(request *dap.EvaluateRequest) {
 			}
 		}
 		// TODO(polina): as far as I can tell, evaluateName is ignored by vscode for expression variables.
-		// Should it be skipped altogether for all levels?
-		exprVal, exprRef := s.convertVariable(exprVar, fmt.Sprintf("(%s)", request.Arguments.Expression))
+		// Should it be skipped alltogether for all levels?
+		exprVal, exprRef := s.convertVariableEvaluate(exprVar, fmt.Sprintf("(%s)", request.Arguments.Expression))
```

Comment:
Indexed paging cannot be supported with expressions?

Reply:
Evaluating an expression does not include support for paging. The request doesn't have context for how many children there may be so it cannot send a request that has a reasonable number. We could possibly parse slice expressions and set the LoadConfig to load the correct number of items, but this could be arbitrarily big defined by the user.

Reply:
I'm sorry you are correct evaluate requests do support paging. I will either make the change or add a TODO.

Reply:
Added support for evaluate.